nasa/harmony
# HARMONY-1468: Add queued badge

## Jira Issue ID
HARMONY-1468

## Description
The Workflow UI work items page has been updated to reflect new work item status of queued.
<img width="868" alt="Screenshot 2023-06-14 at 1 21 07 PM" src="https://github.com/nasa/harmony/assets/34752045/5f8e3b05-6fd9-4be3-977c-95ae95094bc0">

## Local Test Steps
- submit a request
- browse to http://localhost:3000/workflow-ui/your-job-id
- you should (at some point) see a queued status in the table of items and in the list of status filters

## PR Acceptance Checklist
